### PR TITLE
add aria description

### LINKED ISSUE #19415: A11y_SQL Azure Data Studio_NewTable_Cloumns_Labels Or Instructions:NVDA is not announcing the instructions provided by labels.
#### GitHub Tags:
#A11yMAS;#A11yTCS;#Win10;#WCAG3.3.2;#DesktopApp;#A11ySev2;#NVDA;#SH_SQL Azure Data Studio_May2022;#SQL Azure Data Studio;

## Environmental Details:
VS Code: 1.59.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0

## OS Details :
Microsoft Windows 10

## Repro Steps:
1.Add connection to the server with appropriate credentials.
2.Tab till Servers in left navigation Bar.
3.Tab till tables and Press enter
4.Tab till New tables and press enter
5.Tab till column and press enter.
6.Turn on the NVDA.
7.Check the instructions provided by the labels under column properties are announcing by the NVDA or not.

## Note:
Same issue found for all the labels under properties of the Primary key,Foriegn keys,Check constraints,Indexes,General tab items.

## Actual results:
NVDA is not announcing the instructions provided by labels.

## Expected results:
NVDA should announce the instructions provided by the labels.

## User Impact:
Screen reader users not able to understand the purpose of the labels.

## Attachments:
https://user-images.githubusercontent.com/105736242/169036734-a54e85d1-e5fb-469c-b2de-aa9be8188aca.mp4

### PR DESCRIPTION
**Pull request by alanrenmsft**
This PR fixes #19415

### CODE REVIEW

**Review by kisantia — APPROVED** [review]

File: src/vs/base/browser/ui/selectBox/selectBoxCustom.ts [hunk 1]
```diff
@@ -127,6 +127,11 @@ export class SelectBoxList extends Disposable implements ISelectBoxDelegate, ILi
 			this.selectElement.setAttribute('aria-label', this.selectBoxOptions.ariaLabel);
 		}
 
+		// {{SQL CARBON EDIT}}
+		if (this.selectBoxOptions.ariaDescription) {
```
[the remark this answers is not in the document]
Comment (alanrenmsft) [reply src/vs/base/browser/ui/selectBox/selectBoxCustom.ts:131]: this is a little bit different; the other PR is fixing a bug of vscode, but this is adding something new that is not needed by vscode. that is why I am modifying it directly.

[the remark this answers is not in the document]
Comment (Charles-Gagnon) [reply src/vs/base/browser/ui/selectBox/selectBoxCustom.ts:131]: Yeah, but this is a feature that theoretically they could use themselves. 

Can you at least open up a PR in their repo and see if they'll take it? They might not without a clear case of how it'd be used but let's at least try IMO.

**Review by Charles-Gagnon — APPROVED** [review]

File: src/vs/base/browser/ui/selectBox/selectBoxCustom.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (alanrenmsft) [reply src/vs/base/browser/ui/selectBox/selectBoxCustom.ts:131]: @Charles-Gagnon  created a PR: https://github.com/microsoft/vscode/pull/152251

[the remark this answers is not in the document]
Comment (Charles-Gagnon) [reply src/vs/base/browser/ui/selectBox/selectBoxCustom.ts]: Sounds reasonable. Not even sure they'll take it without a clear use case.

**Pull request merged**